epytope/tutorials/data/test.csv

```diff
@@ -1,20 +1,23 @@
-Seq	Method	HLA-B*15:01	HLA-A*02:01
-TVATLISIN	syfpeithi	9	11
-SLANGHDAK	syfpeithi	12	17
-SINTHISLA	syfpeithi	7	19
-SLYNTVATL	syfpeithi	12	31
-YNTVATLIS	syfpeithi	2	4
-VATLISINT	syfpeithi	2	8
-ISINTHISL	syfpeithi	2	16
-TLISINTHI	syfpeithi	13	20
-LISINTHIS	syfpeithi	7	13
-INTHISLAN	syfpeithi	3	3
-LANGHDAKA	syfpeithi	1	17
-HISLANGHD	syfpeithi	6	8
-NTHISLANG	syfpeithi	0	10
-NTVATLISI	syfpeithi	1	18
-ATLISINTH	syfpeithi	2	15
-THISLANGH	syfpeithi	1	5
-ANGHDAKAK	syfpeithi	2	2
-ISLANGHDA	syfpeithi	-1	9
-LYNTVATLI	syfpeithi	2	11
+Allele	HLA-B*15:01	HLA-A*02:01
```

Comment:
This doesn't look right (and should have commas).

Reply:
You are right. This is produced in `line 10` in the `tutorials` in `EpitopePrediction.ipynb`. I would suggest to change the `csv` to a `tsv`. A `tsv` is more visually appealing with a multi-index. What do you think?